test/integration/webhooks/intercom-recap/expected.json

```diff
@@ -17,7 +17,7 @@
       "mentionsUser": [],
       "alertsUser": [],
       "description": "",
-      "status": "open"
+      "status": "closed"
```

Comment:
If I understand correctly, status should be `closed`, because there were no non-whisper messages after last summary.